Get_product_data.py

```diff
@@ -0,0 +1,54 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import os
+
+class Get_product_infor:
+    def __init__(self):
+        self.base_url = "https://www.amazon.co.jp/s?k={}&page={}"
+
+    def get_products_data(self, soup):
+        products = []
+        for product in soup.select("div[data-component-type='s-search-result']"):
+            product_data = {}
+            try:
+                product_data['Name'] = product.find('span', class_='a-size-base-plus a-color-base a-text-normal').text.strip()
```

Comment:
Avoid having too long line. PEP8 suggest 80 characters but usually, 100-char is applied for most of the project because the screen is bigger than before.